tests/date/date.js

```diff
@@ -24,3 +24,9 @@ new Date(2015, 6, 18, 11, 55, '42');
 new Date(2015, 6, 18, 11, 55, 42, '999');
 new Date(2015, 6, 18, 11, 55, 42, 999, 'hahaha');
 new Date('2015', 6);
+
+var b:boolean = d > 0;
+var n1:number = d - 12;
+var n2:number = d & 255;
+var n3:number = d + 12; // expect error
+var s:string = d + 12; // fixme? in js this coerces both to string and concats them
```

Comment:
Assuming the maybe FIXME is that this should throw an error, but does not. Should we file a separate issue for this & reference in the comment?

Reply:
It throws confusing errors, and it possibly should be allowed.